src/svelte/index.ts

```diff
@@ -1,3 +1,3 @@
-import { connect, getActions } from "./components/connect";
-
-export { connect, getActions };
+import { connect, getActions } from "./components/connect";
```

Comment:
This is weird. Prettier made this change, but I can't for the life of me figure out what was modified. There were no changes to whitespace that I can see, but git still detects a change.